config.example.php

```diff
@@ -21,3 +21,6 @@
 
 // Array of boards to not backup regardless of other settings
 $ignore_boards = array('Welcome Board');
+
+// Timestamp format, set to false to disable appending the timestamp
+$ts_format = 'Y-m-d_H-i-s';
```

Comment:
Thanks for the suggestion. Can you set it to false by default? and maybe move the value `'Y-m-d_H-i-s';` in the comment to show an example of how it can be used.

also maybe rename variable to `$filename_append_datetime = false;` ?